core/src/main/java/hudson/cli/DiagnosedStreamCorruptionException.java

```diff
@@ -0,0 +1,55 @@
+package hudson.cli;
+
+import java.io.PrintWriter;
+import java.io.StreamCorruptedException;
+import java.io.StringWriter;
+
+// TODO COPIED FROM hudson.remoting
```

Comment:
So why are we copying it? If it is just not `public` in `remoting`, then file a PR to fix that please…